charts/kube-otel-stack/templates/bridge.yaml

```diff
@@ -0,0 +1,63 @@
+{{ $bridge := $.Values.opampBridge}}
```

Comment:
No need to do this, this is most useful for when you're doing a range through a list

Reply:
i like using less characters in the rest of the file but ill remove it if you think its extra

Reply:
reverted to just use $.Values everywhere since i copied a lot of your other pr